modules/dcache/src/main/java/diskCacheV111/poolManager/CostModuleV1.java

```diff
@@ -83,16 +90,64 @@ public ImmutableMap<String, String> getTagMap() {
         public PoolInfo getPoolInfo() {
             return new PoolInfo(_address, _info, _tagMap);
         }
+
+        public long getSerialId() {
+            return _serialId;
+        }
+
+        public int getTrustScore() {
+            return _trustScore;
+        }
+
+        public boolean getEnabledStatus() {
+            return _enabled;
+        }
+    }
+    public boolean getPoolStatus (String poolName) {
+        return _hash.get(poolName).getEnabledStatus();
+
     }
 
     public synchronized void messageArrived(CellMessage envelope, PoolManagerPoolUpMessage msg) {
+        // CostModuleTest#testPoolCircuitbreaker depends on these vaules beeing as they are.
```

Comment:
Obviously, we don't want test related constants to be a part of the main codebase.